create_kubernetes_sysext.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -euo pipefail
+
+export ARCH="${ARCH-amd64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME CNI_VERSION"
```

Comment:
```suggestion
  echo "Usage: $0 VERSION SYSEXTNAME [CNI_VERSION]"
```
To indicate it's optional